admin/src/common/blocks/MediaGalleryBlock.tsx

```diff
@@ -0,0 +1,41 @@
+import { BlockCategory, createCompositeBlock, createCompositeBlockSelectField, createListBlock } from "@comet/blocks-admin";
+import { MediaGalleryBlockData } from "@src/blocks.generated";
+import { MediaGalleryItemBlock } from "@src/common/blocks/MediaGalleryItemBlock";
+import { aspectRatioOptions } from "@src/common/blocks/StandaloneMediaBlock";
+import * as React from "react";
+import { FormattedMessage } from "react-intl";
+
+const MediaGalleryListBlock = createListBlock({
+    name: "MediaGalleryList",
+    block: MediaGalleryItemBlock,
+    itemName: <FormattedMessage id="mediaGalleryBlock.mediaGalleryList.itemName" defaultMessage="Item" />,
+    itemsName: <FormattedMessage id="mediaGalleryBlock.mediaGalleryList.itemsName" defaultMessage="Items" />,
```

Comment:
```suggestion
    itemName: <FormattedMessage id="mediaGalleryBlock.mediaGalleryList.itemName" defaultMessage="item" />,
    itemsName: <FormattedMessage id="mediaGalleryBlock.mediaGalleryList.itemsName" defaultMessage="items" />,
```

Then it's "Add items" instead of "Add Items", which is more natural.

Reply:
done